TestHelpers/utils.sh

```diff
@@ -19,18 +19,22 @@ install_softhsm() {
 
 development_clevis() {
     # TODO: remove this whole function once the clevis pkcs11 feature development is done
-    rlIsRHEL 9
-    if [ $? -eq 0 ]; then
-        rlRun "dnf install python-pip gcc clang cmake jose libjose cryptsetup socat tpm2-tools luksmeta libluksmeta -y"
+    if rpm -qa | grep -q clevis-pin-pkcs11; then
+        export PKCS11_IMPLEMENTED=1
+        rlLogInfo "Package is already installed on system!"
```

Comment:
I suggest adding info about the installed package. In the logs there is nothing, only a message package is installed.
```
:: [ 16:44:21 ] :: [   INFO   ] :: Package is already installed on system!
```
I would add the outptut of the `rpm -q clevis-pin-pkcs11`